packages/react-components/react-dialog/library/src/components/DialogSurface/useDialogSurfaceStyles.styles.ts

```diff
@@ -39,6 +39,11 @@ const useRootBaseStyle = makeResetStyles({
   boxSizing: 'border-box',
   backgroundColor: tokens.colorNeutralBackground1,
   color: tokens.colorNeutralForeground1,
+  // Same styles as DialogSurfaceMotion last keyframe,
+  // to ensure dialog will be properly styled when surfaceMotion is opted-out
+  boxShadow: tokens.shadow64,
+  transform: 'scale(1) translateZ(0)',
```

Comment:
nit: do we need this style?

Reply:
Ok, just checked, both the transform and opacity can be removed, we only need to add the `boxShadow` 👍🏻